### DISCUSSION TITLE
Why is there html codes above the publication preview image?

### DISCUSSION THREAD

**Discussion by nafis-neehal** [alshedivat/al-folio#2803] [Q&A] [answered]
Why is there HTML codes above the publication preview image? I tried removing it, but couldn't locate the issue. I'm seeing the same issue in the demo site as well (in both firefox and chrome).

 Can anyone please help?

![Screenshot 2024-10-23 at 11 06 14 AM](https://github.com/user-attachments/assets/368ea197-0619-4ff4-9d5a-c604315e3f49)
![Screenshot 2024-10-23 at 11 05 41 AM](https://github.com/user-attachments/assets/30f41d80-37ef-4fb2-b40b-ef962a73e886)

**Comment by george-gca** [comment, marked as answer]
Just found out about this and fixed. Fixed version will be up as soon as [this action](https://github.com/alshedivat/al-folio/actions/runs/11482970478) finishes.

**Comment by nafis-neehal** [comment]
Thank you for your prompt response. What should I do on my end to add this fix to my site (it's already running)?

**Reply by nafis-neehal** [reply]
NVM, found it. Thank you.
